aggregators/config.go

```diff
@@ -34,6 +34,13 @@ type Processor func(
 	aggregationIvl time.Duration,
 ) error
 
+// OverflowLogging defines the logging function when overflow happens,
+// and for which aggregation interval is overflow logging enabled.
+type OverflowLogging struct {
+	Func                func(msg string, fields ...zap.Field)
+	AggregationInterval time.Duration
+}
```

Comment:
[For reviewer] Actually I kind of preferred
```go
type OverflowLogger struct {
    *zap.Logger
    AggregationInterval time.Duration
}
```

But then log level isn't easily configurable.